src/migrations/1602166719410-authentication.js

```diff
@@ -9,6 +9,16 @@ module.exports.up = async function (next) {
     email text UNIQUE,
     password text
   );
+  CREATE TABLE IF NOT EXISTS jobs (
```

Comment:
Should create a new migration for this

Reply:
New migration for jobs table was created.